Test.Utility/TestEnvironmentBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Immutable;
+
+
+namespace Messerli.Test.Utility
+{
+    public class TestEnvironmentBuilder
+    {
+        private readonly ImmutableList<TestFile> _testFiles = ImmutableList.Create<TestFile>();
+
+        public TestEnvironmentBuilder()
+        {
+        }
+
+        private TestEnvironmentBuilder(ImmutableList<TestFile> testFiles) => _testFiles = testFiles;
```

Comment:
Split this onto multiple lines

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/11/commits/7bb17ca31ee65c5013624ed1951fc621ceffa3dd